spec/Candid.md

```diff
@@ -958,24 +961,25 @@ opt <v> : opt <t> ~> null : opt <t'>
 Coercing a non-null, non-optional and non-reserved type at an option type treats it as an optional value, if it can be decoded successfully:
```

Comment:
```suggestion
Coercing a non-null, non-optional and non-reserved type at an option type treats it as an optional value, if it can be decoded successfully at the content type:
```